C/Coin-OR/Clp/build_tarballs.jl

```diff
@@ -1,7 +1,7 @@
 using BinaryBuilder, Pkg
 
 name = "Clp"
-version = v"1.17.5"
+version = v"1.17.3"
```

Comment:
Do you have to update the commit as well? It's `27f61df3a85e327b8f7f15f8f40b0f415f4cc4e9`

Reply:
Thanks! Nice catch.